pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -397,3 +397,13 @@ def test_get_indexer_categorical_time():
          Categorical(date_range("2012-01-01", periods=3, freq='H'))])
     result = midx.get_indexer(midx)
     tm.assert_numpy_array_equal(result, np.arange(9, dtype=np.intp))
+
+
+def test_timestamp_multiindex_indexer():
+    # https://github.com/pandas-dev/pandas/issues/26944
+    dt_index = pd.date_range(start="1jan2019 00:15:33", periods=100,
```

Comment:
Can this be rewritten as 

```
idx = pd.MultiIndex.from_produce([
    pd.date_range("2019-01-01T00:15:33", periods=100, freq="H", nam"date"),
    ['x'],
    [3]
])
df = pd.DataFrame({'foo': np.arange(len(idx))}, idx)
```

I'd find that a bit cleaner than the `reset_index` /  `set_index`.

Reply:
agreed and also was a better date string format than mine.  "1jan2019 00:15:33" was failing on some of the tests as unrecognizable datestring.